xbmc/cores/AudioEngine/Encoders/AEEncoderFFmpeg.cpp

```diff
@@ -290,40 +290,45 @@ int CAEEncoderFFmpeg::Encode(uint8_t *in, int in_size, uint8_t *out, int out_siz
 
     avcodec_fill_audio_frame(frame, channelNum, m_CodecCtx->sample_fmt, in, in_size, 0);
 
-    pkt->size = out_size;
-    pkt->data = out;
-
     /* encode it */
     err = avcodec_send_frame(m_CodecCtx, frame);
     if (err < 0)
       throw FFMpegException("Error sending a frame for encoding (error '{}')",
                             FFMpegErrorToString(err));
 
-    while (err >= 0)
+    err = avcodec_receive_packet(m_CodecCtx, pkt);
+    //! @TODO: This is a workaround for our current design. The caller should be made
+    // aware of the potential error values to use the ffmpeg API in a proper way, which means
+    // copying with EAGAIN and multiple packet output.
+    // For the current situation there is a relationship implicitely assumed of:
+    // 1 frame in - 1 paket out. This holds true in practice but the API does not guarantee it.
```

Comment:
paket -> packet ?

Reply:
Yes. Will close this one and open a new one. Much easier than maintaining my change in another PR.